staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1055,6 +1055,13 @@ const (
 	// in which freePageReporting is always disabled.
 	FreePageReportingDisabledAnnotation string = "kubevirt.io/free-page-reporting-disabled"
 
+	// DeschedulerRequestEvictOnlyAnnotation indicates pods whose eviction is not expected to be completed right away.
+	// Instead, an eviction request is expected to be intercepted by an external component which will initiate the
+	// eviction process for the pod.
+	DeschedulerRequestEvictOnlyAnnotation string = "descheduler.alpha.kubernetes.io/request-evict-only"
+	// DeschedulerEvictionInProgressAnnotation indicates pods whose eviction was initiated by an external component.
+	DeschedulerEvictionInProgressAnnotation string = "descheduler.alpha.kubernetes.io/eviction-in-progress"
```

Comment:
This annotations affect the descheduler and not KubeVirt, so IMO they do not belong in KubeVirt's API.

Reply:
That's right! Thanks